.github/workflows/cobra_commander-ruby.yml

```diff
@@ -5,11 +5,8 @@ on:
 
 jobs:
   ruby:
-    uses: powerhome/power-tools/.github/workflows/_ruby-package.yml@main
+    uses: powerhome/power-tools/.github/workflows/_ruby-workflow.yml@main
     with:
       package: '${{ github.workflow }}'
       workdir: '${{ github.workflow }}'
-      ruby: '["2.7", "3.0", "3.1"]'
```

Comment:
No more testing multiple Ruby versions?

Reply:
The opposite, it tests more now, just the default for all ruby packages: 2.7, 3.0, 3.1, and 3.2.